templates/snippets/header.html

```diff
@@ -49,25 +50,30 @@ <h2 class="my-0 mr-md-auto font-weight-normal" >
             </li> -->
             {% if user.is_authenticated %}
             <li class="nav-item">
-                <a class="btn btn-outline-success mr-sm-3 mb-2 mt-3 my-sm-0" href="{% url 'accounts:password_change' %}">Change Password</a>
+                <a class="btn btn-outline-success mr-sm-3 mb-2 mt-3 my-sm-0"
+                    href="{% url 'home:dashboard' %}">Dashboard</a>
             </li>
 
             <!-- Used in Feedbacks Page -->
             {% if logout_redirect_site %}
             <li class="nav-item">
-                <a class="btn btn-outline-danger" href="{% url 'accounts:logout' %}?next={{ logout_redirect_site }}">Sign Out</a>
+                <a class="btn btn-outline-danger"
+                    href="{% url 'accounts:logout' %}?next={{ logout_redirect_site }}">Sign Out</a>
             </li>
             {% else %}
             <li class="nav-item">
                 <a class="btn btn-outline-danger" href="{% url 'accounts:logout' %}">Sign Out</a>
             </li>
             {% endif %}
-            
+
             {% else %}
+             <li class="nav-item">
+                <a class="btn btn-outline-danger" href="{% url 'home:index' %}">Home</a>
+            </li>
```

Comment:
This button should be like `Dashboard` button in appearance and not like `Sign In` button. You can check the appearance of this button at http://localhost:8000/accounts/password_reset/done/